tony-core/src/test/java/com/linkedin/tony/TestUtils.java

```diff
@@ -40,6 +40,27 @@ public void testUnzipArchive() {
     } catch (IOException e) {
       fail(e.toString());
     }
+  }
 
+  @Test
+  public void testIsArchive() {
+    ClassLoader classLoader = getClass().getClassLoader();
+    File file1 = new File(classLoader.getResource("test.zip").getFile());
+    File file2 = new File(classLoader.getResource("test.tar").getFile());
+    File file3 = new File(classLoader.getResource("test.tar.gz").getFile());
+    assertTrue(Utils.isArchive(file1.getAbsolutePath()));
+    assertTrue(Utils.isArchive(file2.getAbsolutePath()));
+    assertTrue(Utils.isArchive(file3.getAbsolutePath()));
+  }
+
+  @Test
+  public void testRenameFile() throws IOException {
```

Comment:
If this test is run multiple times, will it fail on subsequent attempts, since the `file_move.txt` resource file will have been moved? I think it's better to create a temp file and move that instead.

Reply:
good catch!